alma/controllers/hook/DisplayPaymentReturnHookController.php

```diff
@@ -68,10 +68,41 @@ public function run($params)
             }
         }
 
+        $wording = [
```

Comment:
it is not a PT translation code ... it should in another PR ... is there clickup ticket for that ?

Reply:
I am not sure that adding a `$wording` map in smarty is the best pattern to factorise translations (and I am not sure factorising translations is what you want to do here 😛 ... could you just explain me what is exactly the use case ?)